server/odc-service/src/main/java/com/oceanbase/odc/service/task/schedule/StdJobScheduler.java

```diff
@@ -117,4 +104,44 @@ public void await(Long id, Integer timeout, TimeUnit timeUnit) throws Interrupte
     public EventPublisher getEventPublisher() {
         return configuration.getEventPublisher();
     }
+
+    private void initDaemonJob() {
+        initExpiredCheckJob();
+        initStartPreparingJob();
+    }
+
+
+
+    private void initExpiredCheckJob() {
+        initCronJob("CheckExpiredJob", "CheckExpiredJobGroup",
+                "* 0/1 * * * ?", CheckRunningJob.class);
+    }
+
+    private void initStartPreparingJob() {
+        initCronJob("startPreparingJob", "startPreparingJobGroup",
+                "0/3 * * * * ?", StartPreparingJob.class);
+    }
+
+
+    private void initCronJob(String key, String group, String cronExpression, Class<? extends Job> jobClass) {
```

Comment:
Why not use @Schedule ？

Reply:
there are same impact but quartz is more handy